SimpleWAWS/Service References/Application Insights/ConnectedService.json

```diff
@@ -0,0 +1,7 @@
+﻿{
```

Comment:
so I don't actually think that whole `Service References` folder is needed. You can delete the whole folder.